app/views/learning_partners/_form.html.erb

```diff
@@ -62,7 +49,7 @@
     </div>
      <div>
         <% text = learning_partner.persisted? ? "Update" : "Create" %>
-        <%= form.submit text, class: "btn-default text-white", data: {file_upload_target: "uploadButton"} %>
+        <%= form.submit text, class: "btn-primary text-white", data: {file_upload_target: "uploadButton"} %>
```

Comment:
why do we have data: {file_upload_target: "uploadButton"} here ? class text-white shouldn't  be part of btn-primary class ?

Reply:
No need of  {file_upload_target: "uploadButton"}  ,so removed it and class text-white added to ban-primary class